index.html

```diff
@@ -20,27 +22,160 @@
+        <div class="card-content"></div>
+        <img src="./images/tonic2.png" alt="Tonic" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
 
-    <section class="content">
-      <h1 class="title">
-        I'm Luis <br>Glad to see you!
-      </h1>
-      <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-      <h2 class="connect"> <a href="#">Let's connect</a></h2>
+      <article class="card-4">
+        <div class="card-content"></div>
+        <img src="./images/multipost2.png" alt="Multi-Post Project" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Multi-Post Project</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+    </section>
+    <div class="about">
+      <h2 class="title">
+        About Myself
+      </h2>
+      <p class="descr">Hello! I’m a software developer! I can help you build a product, feature or website. Look through
+        some
+        of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+        contact me.</p>
+      <h3 class="connect"> <a href="#">Let's connect</a></h3>
```

Comment:
- [ ] As previously requested, the styling for the about section is a little offf. Compare these two images, especially the padding.
<div class="flex">
<img src="https://user-images.githubusercontent.com/65501535/133917593-82f80ab5-918d-4b48-924e-a34f9ac2c199.png"  style="width: 50%;"/>
<img src="https://user-images.githubusercontent.com/65501535/133917628-f31fef0d-14a8-41fd-b88a-0828b2c51d5c.png"  style="width: 50%;"/>
</div>